cmd/devdata/system_intake.go

```diff
@@ -140,9 +144,14 @@ func createSystemIntake(
 	requesterName string,
 	requestType models.SystemIntakeRequestType,
 ) *models.SystemIntake {
-	ctx := mock.CtxWithLoggerAndPrincipal(logger, requesterEUAID)
+	var ctx context.Context
+	var requesterEUAIDPtr *string
+	if requesterEUAID != "" {
+		ctx = mock.CtxWithLoggerAndPrincipal(logger, requesterEUAID)
+		requesterEUAIDPtr = &requesterEUAID
+	}
 	// if there's no given intakeID, we can default to the resolver
```

Comment:
I think this comment needs an update based on the additional check added to this `if` statement

Reply:
Good call, addressed in [1aa5882](https://github.com/CMSgov/easi-app/pull/2379/commits/1aa58828dfc88e4c123ea0513eff95ed9363c95a)